docs/devices/TroubleShooting.rst

```diff
@@ -32,21 +32,21 @@ Since 5G support is currently being deployed by mobile phone network operators,
 
 Try multiple SIM cards from different vendors if you experience connectivity issues.
 
-GPS
-***
+GNSS
+****
 
-Error: GPS_EVT_OPERATION_BLOCKED
-================================
+Error: GNSS_EVT_OPERATION_BLOCKED
```

Comment:
```suggestion
Error: NRF_MODEM_GNSS_EVT_BLOCKED
```

Reply:
This event name changed when the GPS module was updated to use the GNSS API in
https://github.com/simensrostad/fw-nrfconnect-nrf/commit/e8488c9d4eb98d7fb877a00bb5043792ea66ed84